tester/pkg/cluster.go

```diff
@@ -0,0 +1,35 @@
+package pkg
```

Comment:
not a fan of package name pkg...should refactor into subdirs of /pkg/ at some point

Reply:
we can find a better directory to put when moving to tester